go/Makefile

```
@@ -24,7 +24,7 @@ pull:
 	@docker compose -f ../deployment/docker-compose.yaml pull
 
 up:
```

checkmake (0.2.2)

[warning] Target body for "up" exceeds allowed length of 5 (12).

(maxbodylength)